src/Rules/FunctionCallParametersCheck.php

```diff
@@ -52,12 +56,14 @@ public function __construct(
 	}
 
 	/**
+	 * @param MethodReflection|FunctionReflection|null $callReflection
 	 * @param Node\Expr\FuncCall|Node\Expr\MethodCall|Node\Expr\StaticCall|Node\Expr\New_ $funcCall
 	 * @param array{0: string, 1: string, 2: string, 3: string, 4: string, 5: string, 6: string, 7: string, 8: string, 9: string, 10: string, 11: string, 12: string, 13?: string} $messages
 	 * @param 'attribute'|'callable'|'method'|'staticMethod'|'function'|'new' $nodeType
 	 * @return list<IdentifierRuleError>
 	 */
 	public function check(
+		$callReflection,
```

Comment:
This breaks larastan: 

```
TypeError: PHPStan\Rules\FunctionCallParametersCheck::check(): Argument #2 ($parametersAcceptor) must be of type PHPStan\Reflection\ParametersAcceptor, PHPStan\Analyser\MutatingScope given, called in /home/runner/work/phpstan-src/phpstan-src/e2e/integration/repo/src/Rules/CheckDispatchArgumentTypesCompatibleWithClassConstructorRule.php on line 133
```

Reply:
Yeah please add the parameters here as last and optional

Reply:
I was just going to ignore it, since it's not `@api`. But OK. 😄